Updates to windows getting started (#21946)
gravitational/teleport: docs/pages/desktop-access/getting-started.mdx

- A Linux server to run the Teleport Desktop Access service on.
  You can reuse an existing server running any other Teleport instance.

> - A server or virtual machine running a Windows Server operating system with

Author: Changed to Windows operating system, good?

> Remote Desktop Services (RDP) enabled and the RDP port available to the Linux server.

Reviewer: Remote Desktop Services generally refers to Windows Server. Let's just say _with Remote Desktop enabled_.

Author: updated

<Admonition type="tip" title="Headless Installation">
The Teleport Windows Auth Setup can be run in a shell environment with elevated
privileges. You can use this to automate installation, uninstallation, and

> certificate updates. The following CLI execution example will install the Teleport Certificate,
> load the required DLL, disable NLA and reboot the Windows machine:

Reviewer 2: Why say lot word when few word do trick? 😆 Also, Oxford comma 4eva.
```suggestion
certificate updates. The following command will install the Teleport Certificate,
load the required DLL, disable NLA, and reboot the Windows machine:
```

certificate updates. For example:

```code

> # Installs Teleport Certificate and reboots

Reviewer 2: It does more than install the certificate. It loads a DLL and disable NLA.

Author: Updated

> # Installs Teleport Certificate, loads required DLL, disables NLA and reboots

Reviewer: This new line isn't code, so it doesn't really belong in the code block.

If you feel a description of what the installer does makes sense, that's fine, but let's write it in text form instead of putting it in a code comment.

Author: Updated.  The big thing was this does a reboot.

(!docs/pages/includes/install-linux.mdx!)

> Create `/etc/teleport.yaml` and configure it for Desktop Access. Update the `proxy_server`
> to your Teleport proxy service and put the Windows machine address under `non_ad_hosts`:

Reviewer 2: ```suggestion
Create `/etc/teleport.yaml` and configure it for Desktop Access. Update the `proxy_server`
value to your Teleport proxy service or cloud tenant, and put the Windows machine address
under `non_ad_hosts`:
```

(!docs/pages/includes/install-linux.mdx!)

Create `/etc/teleport.yaml` and configure it for Desktop Access. Update the `proxy_server`
value to your Teleport proxy service or cloud tenant, and put the Windows machine address
under `non_ad_hosts`:

```yml
version: v3
teleport:

>   nodename: examplewindowsaccess.teleport.com

Reviewer: Since we don't actually own `teleport.com`, let's make sure our examples use subdomains of example.com
```suggestion
  nodename: windows.teleport.example.com
```
